model_prices_and_context_window.json

```diff
@@ -7661,22 +7677,68 @@
         "litellm_provider": "voyage",
         "mode": "embedding"
     },
-    "voyage/voyage-lite-02-instruct": {
-        "max_tokens": 4000,
-        "max_input_tokens": 4000,
-        "input_cost_per_token": 0.0000001,
+    "voyage/voyage-3-large": {
+        "max_tokens": 32000,
+        "max_input_tokens": 32000,
+        "input_cost_per_token": 0.00000018,
         "output_cost_per_token": 0.000000,
         "litellm_provider": "voyage",
         "mode": "embedding"
     },
-    "voyage/voyage-finance-2": {
```

Comment:
same feedback as the last PR - please do not remove the models

Reply:
It's in the PR -- there's just a slight reordering (see my other comment).